packages/runtime/src/index.js

```
@@ -1,47 +1,64 @@
+              elemento.innerHTML = codigoCom
+              if (
+                etiquetas[i].hasAttribute('mode') &&
+                etiquetas[i].getAttribute('mode') == 'modulo'
+              ) {
+                elemento.setAttribute('type', 'module')
+              }
+              document.body.appendChild(elemento)
+            })
+            .catch(function (error) {
+              console.error(error)
+            })
```

Biome (1.9.4)

[error] This function expression can be turned into an arrow function.

Function expressions that don't use this can be turned into arrow functions.
Safe fix: Use an arrow function instead.


(lint/complexity/useArrowFunction)